uint16 count
uint16 start
float32 rate
# TODO
# add a field for the starting value of the counter, preferably unsigned integer
# add a field for the rate of the counter's incrementation, should handle floating point values
# info on types can be found here: http://wiki.ros.org/msg
---
bool result
